File: C/program21.c
#include<stdio.h>
#include<stdbool.h>

int main()
{
    int iValue = 0;
    bool bRet = false;

    printf("Enter number\n ");
    scanf("%d",&iValue);

    bRet = CheckEven(iValue);

    if (bRet == true)
    {
        printf("%d number is Even",iValue);
    }
    else
    {
        printf("%d number is Odd",iValue);
    }

   return 0; 
}

bool CheckEven(int iNO) 
{
    return (iNO % 2 == 0);
}

File: Java/program21.java
import java.util.*;

class Even
{
    public boolean CheckEven(int iNo)
    {
        return ( (iNo % 2) == 0);
    } 
}

class program21
{
    public static void main(String A[])
    {
        int iValue = 0;
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number :");
        iValue = sobj.nextInt();

        Even eobj = new Even();
        bRet = eobj.CheckEven(iValue);

        if(bRet == true)
        {
            System.out.println(iValue +" is Even number");

        }
        else
        {
            System.out.println(iValue+" is Odd number  ");
        }

               

        
    }
}